extract_unique_sensors_01.py

```diff
@@ -0,0 +1,39 @@
+from urllib.request import urlopen
+
+url = ('http://monctl.devm.camlab.kat.ac.za/kat/doc/manuals/sensors/SensorList.txt')
+try:
+    sensors_data = urlopen(url)
+except Exception:
+    filename = ('/home/lebukani/Downloads/SensorList.txt')
+    with open(filename) as lines:
+        sensors_data = lines.readlines()
+print(sensors_data)
+    
+def extract_all_sensors(lines): 
+    """
+    Filters the sensors_data and returns only a list of sensors.
+    
+    """
+    candidate_lines = []
+    for line in lines:
+        if line.startswith('kat.sensors'):
+           for candidate_line in line.split():
+               candidate_lines.append(candidate_line)
+    return candidate_lines
+
+# Loop over candidate_lines and extract sensor names
+def splitting_list():
+    '''
+    Splits the candidate_lines by dot and append that to sensors_list.It also removes the empty lines.
+
+    '''
```

Comment:
- Use “”“triple double quotes”“”
- One-line docstring should fit on one line with quotes